.github/actions/checklist.js

```diff
@@ -0,0 +1,37 @@
+const CHECKLIST_TYPES = {
```

Comment:
Maybe we need ESLint on this file? Same with `pr-title.js`

Reply:
Do we? This file will most likely not be changed again and it's quite coupled to the `actions/github-script@v4` package. I mean, we could but this code is not really part of the projects. Let me know

Reply:
I really couldn't activate it on a `.github` folder. I don't think we should mess around with it since it's not something that will be part of the codebase. It's just actions for Github.